### PR TITLE
feat(quill): add handling for reply and edit messages

### PR DESCRIPTION
**Pull request by phatv**

### CODE REVIEW

File: src/components/ComposerQuill/composer.js [hunk 1]
```diff
@@ -66,9 +84,12 @@ class Composer extends React.Component {
     const {send} = this.props;
 
     return () => {
-      const text = this.quill.root.textContent.trim();
+      const text = this.quill.root.textContent;
```
Comment (robstax) [inline src/components/ComposerQuill/composer.js]: why don't we need `.trim()` anymore?

File: yarn.lock [hunk 2]
```diff
@@ -2115,6 +2115,11 @@ JSONStream@^1.0.4, JSONStream@^1.3.4, JSONStream@^1.3.5:
     jsonparse "^1.2.0"
     through ">=2.2.7 <3"
 
+abab@^2.0.0:
```
Comment (robstax) [inline yarn.lock]: are all these changes supposed to be here?

File: src/components/ComposerQuill/composer.js (hunk 1 above)
Comment (phatv) [reply src/components/ComposerQuill/composer.js]: There was nothing to trim, so it wasn't doing anything

File: yarn.lock (hunk 2 above)
Comment (phatv) [reply yarn.lock]: Hmm, i'm not sure. I don't know where these came from, the only package I added was turndown

Comment (phatv) [reply yarn.lock]: Looks like they're needed for turndown

File: src/components/ComposerQuill/composer.js (hunk 1 above)
Comment (robstax) [reply src/components/ComposerQuill/composer.js]: does it hurt to leave the `trim()` there? seems like maybe there is a scenario where we have trailing whitespace or something?

**Pull request merged**